### PR TITLE
[Tools] Enable PHPCS for CouchDB_Import_Demographics.php

### PR DESCRIPTION
**Pull request by laemtl**
Replaces #6001

### CODE REVIEW

**Comment by pierre-p-s** [comment]
@laemtl 
Travis failed

**Comment by laemtl** [comment]
@pierre-p-s Solved !

File: tools/CouchDB_Import_Demographics.php [hunk 1]
```diff
@@ -158,7 +209,8 @@ function _generateQuery() {
                                 LEFT JOIN participant_status_options pso ON (pso.ID=ps.participant_status)
                                 LEFT JOIN feedback_bvl_thread fbt ON (fbt.CandID=c.CandID) 
                                 LEFT JOIN feedback_bvl_entry fbe ON (fbe.FeedbackID=fbt.FeedbackID)";
-        $groupBy=" GROUP BY s.ID, 
+        //phpcs:enable
+        $groupBy =" GROUP BY s.ID,
```
Comment (pierre-p-s) [inline tools/CouchDB_Import_Demographics.php:213]: Maybe a space after the equal sign to match the syntax

File: tools/CouchDB_Import_Demographics.php [hunk 2]
```diff
@@ -177,182 +229,222 @@ function _generateQuery() {
...
+                                WHERE Name='" . $consentName . "') ";
+                $groupBy       .= ",
                             $cField.Status,
                             $cField.DateGiven,
                             $cField.DateWithdrawn";
-          }
+            }
         }
-        $whereClause=" WHERE s.Active='Y' AND c.Active='Y' AND c.Entity_type != 'Scanner'";
+        $whereClause =" WHERE s.Active='Y' AND c.Active='Y' "
```
Comment (pierre-p-s) [inline tools/CouchDB_Import_Demographics.php:271]: Same here

File: tools/CouchDB_Import_Demographics.php [hunk 3]
```diff
@@ -177,182 +229,222 @@ function _generateQuery() {
...
-                $Dict[$fname]['Type'] = $field['Type'];
+            $consents = \Utility::getConsentList();
+            foreach ($consents as $consent) {
+                $consentName  = $consent['Name'];
+                $consentLabel = $consent['Label'];
+                $this->Dictionary[$consentName]           = [
+                    'Description' => $consentLabel,
+                    'Type'        => "enum('yes','no')",
+                ];
+                $this->Dictionary[$consentName . "_date"] = [
```
Comment (pierre-p-s) [inline tools/CouchDB_Import_Demographics.php:324]: Add 2 tabs line 320 ad 324 to match the equal sign line 328

**Comment by pierre-p-s** [comment]
I ran phpcs and did not get an error. Just 3 minor comments on the code

File: tools/CouchDB_Import_Demographics.php (hunk 3 above)
Comment (pierre-p-s) [reply tools/CouchDB_Import_Demographics.php:324]: Nvm this makes phpcs fail.

**Comment by laemtl** [comment]
@driusan Ready for review.

**Comment by pierre-p-s** [comment]
phpcs worked and comments were addressed

**Pull request merged**